testsuite/features/profiles/Docker/authprofile/sles15sp4.repo

```diff
@@ -0,0 +1,6 @@
+[sles15sp4]
+name=sles15sp4
+enabled=1
+autorefresh=0
+baseurl=http://download.suse.de/download/ibs/SUSE/Products/SLE-Product-SLES/15-SP4/x86_64/product/
```

Comment:
I am wondering why there is an extra `download` before `ibs` in the URL compared to the SLE12 one:
![image](https://user-images.githubusercontent.com/12104291/167427396-3800559e-c85e-4e97-9a34-6a65d8bf1a1b.png)

Reply:
Actually, both are working, so I will remove the extra folder